src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4066,10 +4066,10 @@ public override bool PreVisit(CallNode node)
                     return false;
                 }
 
-                // If there are no overloads with lambdas, we can continue the visitation and
+                // If there are no overloads with lambdas or identifiers, we can continue the visitation and
                 // yield to the normal overload resolution.
-                var overloadsWithLambdas = overloads.Where(func => func.HasLambdas);
-                if (!overloadsWithLambdas.Any())
+                var overloadsWithLambdasOrIdentifiers = overloads.Where(func => func.HasLambdas || func.HasIdentifiers);
```

Comment:
As mentioned in the teams thread, I am concerned about functions which take lambdas also taking identifiers in the exact same argument slot. Right now, this is not the case. However, we need to add asserts to make absolutely sure.

Reply:
Added new code in PowerFxConfig AddFunction method to test this + extra UTs